start/env.ts

```diff
@@ -34,4 +34,9 @@ export default await Env.create(new URL("../", import.meta.url), {
   DB_USER: Env.schema.string(),
   DB_PASSWORD: Env.schema.string.optional(),
   DB_DATABASE: Env.schema.string(),
+
+  // Custom variables
+  MENU_URL: Env.schema.string(),
+  USERS_URL: Env.schema.string(),
```

Comment:
tutaj już chyba możesz zrobić
```suggestion
  MENU_URL: Env.schema.string({ format: "url" }),
  USERS_URL: Env.schema.string({ format: "url" }),
```